internal/store/postgres/asset_repository.go

```diff
@@ -775,6 +776,7 @@ func (r *AssetRepository) getAssetSQL() sq.SelectBuilder {
 		a.service as service,
 		a.description as description,
 		a.data as data,
+		COALESCE(a.url, '') as url,
```

Comment:
how this compared to setting url default to '' when altering table?

Reply:
Altering a table with default value used to take a lock for updating all rows (which would block reads and writes). This looks to have been fixed in PG 11 but I wasn't aware of that while making these changes.